spec/dummy/client/webpack.server.rails.build.config.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
     filename: 'server-bundle.js',
 
     // This needs a leading slash.
-    publicPath: '/' + webpackPublicOutputDir,
+    publicPath: '/' + webpackPublicOutputDir + '/',
```

Comment:
This was a change @robwise and I made bc not all leading slashes were added in all the right places. (Only in images). Thus, we removed the leading slashes before images in webpack.common.config.js